pkg/debug/Dockerfile

```diff
@@ -1,9 +1,6 @@
-FROM alpine:3.12
-ENTRYPOINT []
-WORKDIR /
+FROM linuxkit/sshd:666b4a1a323140aa1f332826164afba506abf597
 
-RUN apk add --no-cache pciutils usbutils vim tcpdump gdb valgrind perf util-linux
-
-COPY debug-spin.sh /
-
-CMD ["/debug-spin.sh"]
+# Feel free to add additional packages here, but be aware that
+# EVE's rootfs image can be no larger than 300Mb
+# RUN apk add --no-cache gdb valgrind
+RUN apk add --no-cache pciutils usbutils vim tcpdump perf
```

Comment:
Can we add `openssh-client` as well? That will make the `ssh` CLI available inside the container, but also `scp`, which is necessary to copy things off (e.g. log files).

Reply:
That's already included in the linuxkit container we're basing this off of.